explorer/client/src/__tests__/reducers/config.test.ts

```diff
@@ -0,0 +1,33 @@
+import reducer, { IState } from '../../reducers'
+import { Action } from '../../reducers/config'
+
+describe('reducers/config', () => {
+  it('returns an initial state', () => {
+    const action = {} as Action
+    const state = reducer({}, action) as IState
+
+    expect(state.search).toEqual({
+      etherscanHost: undefined
+    })
+  })
+
+  it('can update the search query', () => {
+    const action = {
+      type: 'UPSERT_JOB_RUN',
+      data: {
+        meta: {
+          jobRun: {
+            meta: {
+              etherscanHost: 'ropsten.etherscan.io'
```

Comment:
Do we have to have so many `meta` nested in the payload?

```
data: { meta: { etherscanHost: ...
```

The above wouldn't work?

Reply:
I'm not a fan of the structure either. It's generated by [`normalize(...)`](https://github.com/yury-dymov/json-api-normalizer)